api/src/main/java/org/openmrs/module/initializer/api/c/ConceptsLoader.java

```diff
@@ -12,4 +14,8 @@
 	public void setParser(ConceptsCsvParser parser) {
 		this.parser = parser;
 	}
+	
+	@Override
+	protected void preload(File file) {
+	}
```

Comment:
Concepts are not subject to the displays pre-loading since they are i18n'd already.